career/linkedin.md

```diff
@@ -0,0 +1,44 @@
+### Why is LinkedIn important?
+
+Recruiters use LinkedIn to connect with candidates. Your CV highlights your career at a glance, LinkedIn paints a portrait of you as a web developer. The LinkedIn’s community is very active, with many contributors, this means that LinkedIn gives you the chance to create a great personal brand
+
+
+##### Your goals with LinkedIn are to:
+
+- Show up in a search in the recruiter's search result with the right keywords
+- Signal that you are qualified and motivated
+
+### Show Up in Search Results
+
+In order to show up in the search results, you need to add the right keywords to your LinkedIn “summary”. The right keywords are often technologies and buzzwords that are commonly used to describe a certain field within technology like “FinTech” or “MedTech” and so on. Add the keyword describing the field that you would like to work in or have experience with. This will make sure that you are found in the recruiter's searches. Do remember to fill in information about your experience in the summary field as well. The summary field is the first thing that the recruiters look at so make sure that you are well presented. Be direct and to the point.
+
+When recruiters do find your profile you need to stand out in the search results. This means that you should have a good headline. The headline is the information visible directly underneath your name in the search results. You should try to add a keyword to your profile page along with the benefit you could provide to a company. Be professional and avoid silly headlines. You want to stand out from the crowd, but for the right reasons.
+
+### Signal that you are qualified for the job
+
+Add your previous work experience even though if it's not directly related to tech. Experience in different fields is a big advantage in many fields, not only tech. Make sure that you point that out in your profile.
+Add the certificates and courses you have taken. Make sure that you clarify what you learned during these courses. And how you applied them.
+
+### Make your profile more complete
+
+LinkedIn has a feature that allows you to view the level of completeness of your profile. This means that LinkedIn shows you if there are sections in your profile that you are missing or that you could add more content to. But just in case. Make sure that you fill out all the sections in your profile.
```

Comment:
`But just in case. Make sure that you fill out all the sections in your profile.`
`But just in case - make sure that you fill out all the sections in your profile.`

Reply:
Done 👍